pkg/kine/logstructured/sqllog/sql.go

```diff
@@ -565,6 +566,19 @@ func (s *SQLLog) Create(ctx context.Context, key string, value []byte, lease int
 	return rev, nil
 }
 
+func (s *SQLLog) Update(ctx context.Context, key string, value []byte, prevRev, lease int64) (rev int64, updated bool, err error) {
+	rev, updated, err = s.d.Update(ctx, key, value, prevRev, lease)
+	if err != nil {
+		return 0, false, err
+	}
+
+	select {
```

Comment:
maybe we can refactor this select blcok into a `s.maybeNotify(rev)` or `s.tryNotify(rev)`?

Reply:
We're hoping to remove this implementation when improving the watchers. 
Since we have this call in a few places we could consider making it a separate function.
FYI, currently this wakes up the watcher poll loop to check for the change to the database.